monitoring/promctl/main.go

```diff
@@ -299,6 +310,9 @@ func main() {
 	RootCmd.PersistentFlags().StringVar(&host, "host", defaultHost, "host")
 	RootCmd.PersistentFlags().Int32VarP(&period, "period", "p", defaultPeriod, "reload period in minute")
 
+	if err := addExternalURL(&prometheusArgs); err != nil {
```

Comment:
Since your function `addExternalURL` doesn't return an error, it seems unnecessary to have a function here. Something like this should be enough:
```
// Set Prometheus external URL if provided
	if url := os.Getenv(externalURLEnv); url != "" {
		prometheusArgs = append(prometheusArgs, fmt.Sprintf("%s=%s", externalURLOption, url))
	}
```